app/views/chomp_sessions/_timer.html.erb

```diff
@@ -1,6 +1,36 @@
 <div class="timer">
-  <p class="time-desc time-desc-title">
-  <span><%= ((@chomp_session.updated_at + @chomp_session.session_expiry.hour - Time.now)/3600).floor %>H
-  <%= ((@chomp_session.updated_at + @chomp_session.session_expiry.hour - Time.now)% 3600 /60).floor %>Min</span> before poll closes</p>
-  <div id="clock"></div>
+  <div class="time-desc time-desc-title w-100">
```

Comment:
Might want to add some space in-between the timer and the text

Reply:
i will add it in another branch ba